### PR TITLE
Add units to display

### PR DESCRIPTION
**Pull request by LaPommeCosmique**
Modified display to show vehicle speed instead of angular acceleration

### CODE REVIEW

File: src/main.cpp
Comment (LaPommeCosmique) [inline src/main.cpp:1]: I added units + another line to the display - but I could only guess the positions of the texts (so their position may need to be changed)

Comment (cosparks) [reply src/main.cpp:1]: I think you may have guessed right!

**Comment by cosparks** [comment]
I can't help but think that this should just be part of your other PR, as they both touch on the same code quite a lot.  Would you mind just combining them?  We could close this PR and you just push these changes to your other branch

File: src/Sensor/SensorOptical.cpp [hunk 1]
```diff
@@ -52,3 +57,8 @@ float SensorOptical::getAngularVelocity() {
     float ret = ((float)n / NUM_APERTURES) * 2 * PI * (MEGA / (float)deltaT);
     return ret;
 }
+
+float SensorOptical::getLinearVelocity() {
+    float ret = VelocityFactor * getAngularVelocity();
```
Comment (cosparks) [inline src/Sensor/SensorOptical.cpp:62]: the fact that computing angular velocity modifies the state of SensorOptical is problematic, say if we want to record angular and linear velocity readings.  I made some comments in your other PR for the direction I was planning on taking this class

**Pull request closed without merging**